site/layouts/shortcodes/gs-prerequisites.html

```diff
@@ -2,7 +2,7 @@
 {{- $test := eq (.Get "link_test") "true" | default true }}
 <p>The following prerequisites are required to create a GameServer:</p>
 <ol>
-<li>A Kubernetes cluster with the UDP port range 7000-8000 open on each node.</li>
+<li>A Kubernetes cluster with the UDP port range 7000-8000 open on each node. <a href="https://agones.dev/site/docs/installation/creating-cluster/gke/#creating-the-firewall">Creating the firewall</a></li>
```

Comment:
This should resolve to either the local or which snapshot version you are seeing - so it can't point at the domain directly.

You can use:
https://github.com/googleforgames/agones/blob/772fffbfaf8a3bdb0cfc1505b6ed51521f4e95d8/site/config.toml#L15

As the root of the link, via:

https://gohugo.io/templates/shortcode-templates/#:~:text=Params.shortcode_color).-,%24.Page.Site.Params,-refers%20to%20parameters

Reply:
Okok sg, i used /site/ now inside the href

Reply:
I think what @markmandel is saying is that even `baseURL` could be variable, so you need to use a function here to grab the config.

@markmandel any reason we can't use https://gohugo.io/methods/site/baseurl/ here? I think this just becomes `{{ .Site.BaseURL }}docs/installation/creating-cluster/gke/#creating-the-firewall`.